test/unit/calculators/statutory_sick_pay_calculator_test.rb

```diff
@@ -1031,23 +1031,61 @@ class StatutorySickPayCalculatorTest < ActiveSupport::TestCase
             assert_equal true, calc.valid_period_of_incapacity_for_work?
           end
 
-          should "not pay SSP if they're self-isolating due to someone they live with" do
+          should "pay SSP if they're self-isolating due to someone they live with" do
             calc = StatutorySickPayCalculator.new(
               sick_start_date: start_date,
               sick_end_date: end_date,
               days_of_the_week_worked: %w(1 2 3 4 5),
               has_linked_sickness: false,
               coronavirus_related: true,
               has_coronavirus: false,
-              cohabitant_has_coronavirus: false,
```

Comment:
I find it odd that this was ever `false`, given the test description! Was it because people used to self-isolate even if a cohabitant didn't have coronavirus?

Reply:
Quite honestly I didn't understand what was going on with this test either, originally it basically amounted to "person is self-isolating due to coronavirus, neither they nor their co-habitant have coronavirus", So I was very confused, so I changed it to a test that actually made sense to me.